website/content/docs/concepts/variables.mdx

```diff
@@ -74,7 +74,7 @@ The available capabilities for Variables are as follows:
 
 ## Task Access to Variables
 
-In Nomad 1.4.0 tasks can only access Variables with the [`template`] block. The
+Tasks can only access Variables with the [`template`] block. The
```

Comment:
Removing the version here changes the meaning of "only" here in a way that's no longer true. In Nomad 1.5.0+, tasks can also access Variables via the Task API.

Reply:
Oh true, I forgot that one, fixed